openshift/server.dc.yml

```diff
@@ -119,24 +119,18 @@ objects:
                       key: sa-password
                       name: ${APP_NAME}-keycloak
                 - name: MONGO_HOST
-                  value: ${APP_NAME}-mongodb
+                  value: ${APP_NAME}-mongo-headless
```

Comment:
Do we have more than 1 mongo name? Why are we specifying headless in the name?

Reply:
hcap-mongo-headless is the name of the service that helm spins up.